### PR TITLE
Adding scaling-group repo to peribolos

### PR DESCRIPTION
**Pull request by jwcesign**
Adding `scaling-group` sandbox repo to peribolos

Part of https://github.com/knative/community/issues/1195

### CODE REVIEW

**Comment by jwcesign** [comment]
cc @dprotaso @psschwei @evankanderson

File: peribolos/knative-sandbox-OWNERS_ALIASES [hunk 1]
```diff
@@ -249,6 +249,11 @@ aliases:
   - kvmware
   - psschwei
   - upodroid
+  scaling-group-approvers:
+  - dprotaso
+  - evankanderson
+  - jwcesign
+  - psschwei
```
Comment (dprotaso) [inline peribolos/knative-sandbox-OWNERS_ALIASES:256]:
```suggestion
  scaling-group-approvers:
  - jwcesign
```

I think it just makes sense to add yourself


In the OWNER file for the repo you can include @psschwei and myself via the `serving-wg-leads`
https://github.com/knative/community/blob/7bd466197944e39503c107ac34503850c67cfa9a/peribolos/knative-sandbox-OWNERS_ALIASES#L268

Then add TOC as a fallback - ie see serving's [OWNERS ](https://github.com/knative/serving/blob/main/OWNERS#L3-L8 )file

**Comment by dprotaso** [comment]
Also @jwcesign is there anyone else you want to add as a reviewer/approver?

File: peribolos/knative-sandbox-OWNERS_ALIASES (hunk 1 above)
Comment (jwcesign) [reply peribolos/knative-sandbox-OWNERS_ALIASES]: fixed

**Comment by jwcesign** [comment]
> Also @jwcesign is there anyone else you want to add as a reviewer/approver?

I added, thanks for reminding.

**Review by psschwei — COMMENTED** [review]
/approve

**Pull request merged**